CONTRIBUTING.md

```diff
@@ -201,7 +201,7 @@ follow the Carbon documentation and coding styles.
 
     -   New features should have a documented design that has been approved
         through the [evolution process](docs/project/evolution.md). This
-        includes modifications to pre-existing designs.
+        includes modifications to preexisting designs.
```

Comment:
This seems to make sense since it's a dictionary word. Plus https://developers.google.com/style/word-list (which we use for doc style) seems to err in this direction with pre-defined, etc.